web/pgadmin/utils/aws_iam.py

```
@@ -0,0 +1,110 @@
+    import botocore.session
+    from botocore.exceptions import BotoCoreError, ClientError, \
+        NoCredentialsError, PartialCredentialsError
+    BOTOCORE_AVAILABLE = True
+except ImportError:
+    BOTOCORE_AVAILABLE = False
+
+_ = gettext
+
+
+def generate_rds_auth_token(host, port, username, region, profile=None,
+                              role_arn=None):
```

Ruff (0.14.14)

[warning] Unused function argument: `role_arn`

(ARG001)